packages/network-enablement-controller/jest.config.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+const merge = require('deepmerge');
+const path = require('path');
+
+const baseConfig = require('../../jest.config.packages');
+
+const displayName = path.basename(__dirname);
+
+module.exports = merge(baseConfig, {
+  // The display name when running multiple projects
+  displayName,
+
+  // An object that configures minimum threshold enforcement for coverage results
+  coverageThreshold: {
+    global: {
+      branches: 88.47,
+      functions: 97.43,
+      lines: 94.47,
+      statements: 94.29,
+    },
+  },
+
+  // Currently the tests for NetworkController have a race condition which
+  // causes intermittent failures. This seems to fix it.
+  testEnvironment: 'jsdom',
```

Comment:
Nit: this seems to be an issue with `NetworkController` do we have the same issue with `NetworkEnablementController` ?
```suggestion